bundles/org.openhab.binding.revogi/src/main/java/org/openhab/binding/revogi/internal/api/DiscoveryRawResponse.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.revogi.internal.api;
+
+import java.util.Objects;
+
+/**
+ * @author Andi Bräu - Initial contribution
+ */
+public class DiscoveryRawResponse {
```

Comment:
You could move the DTOs into a package called `dto` to avoid warnings about the missing NonNullByDefault annotation.